web/js/map-OpenLayers.js

```diff
@@ -247,18 +259,18 @@ $.extend(fixmystreet.utils, {
        * the spinner in the DOM.
        */
       loading_spinner: {
-          count: 0,
+          count: {},
           show: function() {
-              fixmystreet.maps.loading_spinner.count++;
-              if (fixmystreet.maps.loading_spinner.count > 0) {
+              fixmystreet.maps.loading_spinner.count[this.id] = 1;
```

Comment:
Previously `fixmystreet.maps.loading_spinner.show` could be called from anywhere, but this change requires it to be called in the context of an OpenLayers event due to `this.id`. Not necessarily a problem, as it is currently only called when an event has fired, but worth noting this implicit coupling.

This could be worked around by using `fixmystreet.maps.loading_spinner.show.call({id: 'test'})`, or something similar.